xtask/src/codegen/aya_bpf_bindings.rs

```diff
@@ -46,7 +46,7 @@ pub fn codegen(opts: &Options) -> Result<(), anyhow::Error> {
             .constified_enum("BPF_FLOW_.*");
 
         let types = [
-            "bpf_map_.*",
+            "bpf_.*",
```

Comment:
Further down we whitelist `bpf_adj_room_mode` which I think can be removed now?

Reply:
Yes, you are correct! Thank you. Updated.